src/utils/cache.rs

```diff
@@ -54,7 +57,10 @@ impl Cache for ServiceCache {
     }
 
     fn invalidate_pattern(&self, pattern: &str) {
-        pipeline_delete(self, self.scan_match(pattern).unwrap());
+        pipeline_delete(
+            self,
+            scan_match_count(self, pattern, redis_scan_count()).unwrap(),
```

Comment:
The `unwrap` could be moved to `scan_match_count` or not... I was thinking yesterday if it was good to panic here (I mean in this module in general). Probably yes, right?